sdk/core/azure-core-management/src/main/java/com/azure/core/management/annotations/AzureHost.java

```diff
@@ -21,19 +21,21 @@
  *
  * Example 1: Azure Resource Manager
  *
- *   {@literal @}AzureHost(AzureEnvironment.Endpoint.RESOURCE_MANAGER)
- *   interface VirtualMachinesService {
- *     {@literal @}GET("subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{vmName}")
- *     VirtualMachine getByResourceGroup(@PathParam("resourceGroupName") String rgName, @PathParam("vmName") String vmName, @PathParam("subscriptionId") String subscriptionId);
- *   }
+ * {@literal @}AzureHost(AzureEnvironment.Endpoint.RESOURCE_MANAGER)
+ * interface VirtualMachinesService {
+ * {@literal @}GET("subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft
+ * .Compute/virtualMachines/{vmName}")
+ * VirtualMachine getByResourceGroup(@PathParam("resourceGroupName") String rgName, @PathParam("vmName") String
```

Comment:
Will this show up correctly? I don't see that `{@literal @}` pattern you used.

Reply:
it was like that already XD.  just had it wrapped in 2 lines and make sure there's no javadocs errors...  
Probably worth it to create an issue out of this to address it

Reply:
Yeah. Feel free to create an issue.